### PR TITLE
27 add sound effects

### PR DESCRIPTION
**Pull request by JudeKlatte**
Added various sound effects for things in game using the website below. Change them with your own sounds as you please, they're not all perfect. 

https://sfxr.me/

### CODE REVIEW

**Comment by jvgusch** [comment]
I would really recommend making the function names all similar. Right now, you have a lot of different function names like `deepClickSound()`, `alVoice()`, and `Explosion()` that isn't very descriptive. I'm not sure what `alVoice()` does just by looking at it. I'd recommend doing something like `playAlVoiceSound()` and `playExplosionSound()` etc. Then it's very clear you are playing a sound.

File: frontend/src/soundEffect.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const soundEffects = {
```
Comment (jvgusch) [inline frontend/src/soundEffect.ts:1]: Consider making this an enum rather than an object.

**Review by jvgusch — APPROVED** [review]

File: frontend/src/soundEffect.ts (hunk 1 above)
Comment (MangoMagoCane) [reply frontend/src/soundEffect.ts:1]: Can typescript enums be indexed with variables?

Comment (MangoMagoCane) [reply frontend/src/soundEffect.ts:1]: I guess so! It's kind of weird how JS objects are both hashmaps/dicts and JS' way of doing inheritence.
image

**Review by jvgusch — DISMISSED** [review]

**Review by kavyab2006 — APPROVED** [review]

**Review by emadsu — APPROVED** [review]
bugs bugs go away plague our code another day

**Review by kavyab2006 — APPROVED** [review]

**Pull request merged**